subjects/ai/emotions-detector/README.md

```diff
@@ -1,62 +1,148 @@
 # Emotions detection with Deep Learning
```

Comment:
the format of the subject is a bit odd if we set up in a full screen it looks odd, half screen wold look good here is an example in half:
![image](https://github.com/01-edu/public/assets/53977751/6969fc35-6d11-41a5-9af7-a08b345549a6)

And in full (looks empty):
![image](https://github.com/01-edu/public/assets/53977751/71d75f23-aad1-4914-b610-2ba4610ea1c2)


the links get broken also
![image](https://github.com/01-edu/public/assets/53977751/ff569e2f-076a-4656-a597-b2b9ccdc6025)